tools/container/hue/supervisor-files/hue_loglistener_template

```diff
@@ -1,6 +1,6 @@
 [program:hue_loglistener]
 directory=${HUE_HOME}
-command="${HUE_HOME}/desktop/core/src/desktop/loglistener.py"
+command=python3.8 ${HUE_HOME}/desktop/core/src/desktop/loglistener.py
```

Comment:
Is this code going to produce an error in Python 3.9 or not?
@athithyaaselvam @wing2fly @ranade1